### ISSUE TITLE
Remove all unused material

### ISSUE THREAD

**Issue by svenvanderburg** [esciencecenter-digital-skills/scikit-learn-mooc#2]
This is the material that we will use:

1:00 Machine learning concepts: Use the slides from the course: [https://inria.github.io/scikit-learn-mooc/ml_concepts/slides.html ](https://inria.github.io/scikit-learn-mooc/ml_concepts/slides.html%E2%80%AF) but add: 

distinction between AI, ML, DL 

Overview of all model families based on first slides of [this presentation,](https://nlesc.sharepoint.com/:p:/s/instructors/EZ38_gcWZ7FIisZBx6UwwgkBB90qmyIJXGIYrl9S6Yrcbw?e=BuxsmQ) (needs to be edited a bit). Don’t go into details we will go into details in the course. 

0:30 [Predictive modeling pipeline: data exploration](https://inria.github.io/scikit-learn-mooc/python_scripts/01_tabular_data_exploration.html)  

Focus on the bare essentials [@Flavio Hafner](mailto:f.hafner@esciencecenter.nl)  

First, emphasize that a good understanding of the data is important for modeling: make a demo (no livecoding, but show plots either in notebook or with some slides) with the most important plotting approaches (`df.hist` and `sns.pairplot`); discuss imbalance in features and in targets (with `df[column_name].value_counts()`) 

Second, let the students do the exercise on data exploration with the penguins dataset (follow the material) 

3:30 Predictive modeling pipeline: fitting a model on numerical data (should be splitted over different instructors) 

First model with scikitlearn 

Working with numerical data 

Preprocessing for numerical features: 

 Skip plotting scaled features 

Skip timing the training, they just have to believe that it is faster 

Validation of a model 

0:15 Intuitions on linear models (https://inria.github.io/scikit-learn-mooc/linear_models/linear_models_intuitions_index.html )  Note: skip under & overfitting 

 

1:30 Predictive modeling pipeline: Handling categorical data, encoding categorical variables 

0:15 Intuitions on tree-based models (https://inria.github.io/scikit-learn-mooc/trees/trees_intuitions_index.html )  Note: skip under & overfitting 

0:30 Predictive modeling pipeline: Combining numerical and categorical data. Skip the Histgradientboosting classifier part because it is weird to use it without introducing it. A different exercise instead of [the exercise about decision trees being faster when encoding is ordinal,](https://inria.github.io/scikit-learn-mooc/python_scripts/03_categorical_pipeline_ex_02.html#exercise-m1-05) but instead do an exercise that is more about general understanding of the ML pipeline. 

0:30 Theory on selecting the best model: [https://inria.github.io/scikit-learn-mooc/overfit/overfit_module_intro.html ](https://inria.github.io/scikit-learn-mooc/overfit/overfit_module_intro.html%E2%80%AF) 

(0:15) Under & overfitting (no live coding) 

(0:15) Learning curves (no live coding) 

Introduce more advanced topics such as: model sweep, hyperparameter tuning, automated feature selection, explainable machine learning: feature importance, no free lunch theory. Based on 2nd part of [this presentation,](https://nlesc.sharepoint.com/:p:/s/instructors/EZ38_gcWZ7FIisZBx6UwwgkBB90qmyIJXGIYrl9S6Yrcbw?e=BuxsmQ) 

1:00 Try to improve accuracy on US Census dataset by applying more advanced strategies. 

0:30 Concluding remarks: You can use [these slides.](https://github.com/esciencecenter-digital-skills/2023-11-28-ds-sklearn/blob/main/files/sklearn%20scikitlearn%20slides%20concluding%20remarks.pptx) Which is a selection from https://inria.github.io/scikit-learn-mooc/concluding_remarks.html plus some best practices

**Comment by svenvanderburg** [comment]
Fixed with #14
